content/vagrant-local-for-k8s.md

```diff
@@ -0,0 +1,193 @@
+# Vagrant never gets old - 
+
+TODO: fancier title
+
+Vagrant is not new. In fact, it's so not new that articles providing a good entry level overview as well as the most important tips and tricks have become hard to come by. It's not one of those new shiny objects anymore, so there's no point in writing that article in 2021, right? 
+
+Vagrant continues to be an extremely useful tool though. So let's get back to it for a short while and have a look what it has to offer. Now, you could figure all this out by just reading the documentation and playing around a bit. But if you're trying to get a job done (that does not happen to be learning Vagrant in the first place), you might find this article helpful. 
+
+# What is Vagrant?
+
+In a nutshell, Vagrant enables you to define virtual machines in a [declarative](https://en.wikipedia.org/wiki/Declarative_programming) way. It gives you a way to keep the configuration of your VM, e.g. how much memory, disk and CPUs it needs, as well as any possible post install provisioning and configuration steps in a file, so that these become easily reproducible. You'd otherwise need to perform manually each time you spin up a new VM off of an installation image, and over time it would become tricky to get exactly the same result twice.
+
+In addition, Vagrant also gives you a convenient way to start, suspend, resume and update your VMs without having to interact with your hypervisor directly. 
+TODO: walk through the steps top spin up a VM without Vagrant to illustrate the difference? 
+
+In a nutshell, think of it as a wrapper around your hypervisor. 
+
+TODO: Draw a parallel to libvirt? Maybe put this in a "Other tools" section towards the end...
+
+# The basics
+
+## Installing Vagrant
+
+Step 1: make sure your hypervisor is set up. 
+Step 2: install vagrant
+    - MacOS: use homebrew
+    - Windows: use installer from Hashicorp or Windows package manager
+    - Linux: your distro's package manager, or upstream package (deb or rpm depending on what distro you use) 
```

Comment:
Maybe call out `zypper in vagrant` for opensuse specifically?

Reply:
Oh yes, that part was a placeholder actually. Need to do another commit.